apps/mobile-wallet/src/components/AddressBadge.tsx

```diff
@@ -53,10 +53,10 @@ const AddressBadge = ({
       {address ? (
         <AddressBadgeContainer>
           {!hideSymbol && <AddressColorSymbol addressHash={addressHash} size={16} />}
-          {address.settings.label ? (
+          {address.label ? (
```

Comment:
The address settings in the desktop wallet are stored flat in the `Address` type, so I changed this in the mobile wallet as well to prepare for the migration.

Reply:
sounds reasonable